unfoldingWord/translationCore
# Sandbox feature usfm3

#### This pull request addresses:

Lots of USFM changes from importing to exporting to alignment data, and usage in tools.

#### How to test this pull request:

Requires checking out branches from `translationCore`, `ScripturePane` and `WordAlignmentTool`.
Test the entire app make sure all works as intended.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3764)
